lib/test_suite.py

```diff
@@ -125,10 +125,10 @@ def __init__(self, suite_path, args):
             self.ini[i] = dict.fromkeys(self.ini[i].split()) \
                 if i in self.ini else dict()
         for i in ["lua_libs"]:
-            self.ini[i] = map(
-                lambda x: os.path.join(suite_path, x),
-                dict.fromkeys(self.ini[i].split())
-                if i in self.ini else dict())
+            lua_libs = self.ini.get(i, "")
+            self.ini[i] = []
+            for p in lua_libs.split():
+                self.ini[i].append(os.path.join(suite_path, p))
```

Comment:
It is not necessary if we'll use 'fork' multiprocessing start method. However, it seems, it is better to keep 'fork' (if'll leave my investigations in another comment).

Reply:
Agree, it was a part of another attempt to fix. Removed.